.devilbox/www/htdocs/mail.php

```diff
@@ -174,7 +174,13 @@
 								<tr id="mail-<?php echo $data['num'];?>" style="display:none">
 									<td></td>
 									<td colspan="4">
-										<?php echo $structure->body ?>
+										<?php
+										if (isset($structure->parts)) {
+											echo $structure->parts[0]->body;
```

Comment:
Can there be more than the 0<sup>th</sup> element?

Reply:
I had 2 in my tests, but they were the same so I picked the first. Would be nice to find a resource that has the different ways an email is sent.